### PR TITLE
LinkDBQuery for Jobs Page

### PR DESCRIPTION
**Pull request by hxghhhh**
- All the core.careers page to query jobs from DB made by Admin

### CODE REVIEW

**Comment by hxghhhh** [comment]
@ssk13 @DannyRL

File: modules/core/client/views/careers.client.view.html [hunk 1]
```diff
@@ -14,21 +14,28 @@ <h5 class="header col s12 light">Available Positions</h5>
...
+
+<!--   Job List  -->
+  <div class="row">
+    <ul class="jobsList">
+      <li class="jobList-item" data-ng-repeat="job in Jobs">
+        <div class="collapsible-header" ng-click="showBody1 === 'false' ? showBody1 = 'true' : showBody1 = 'false'"><i class="material-icons">games</i>{{job.title}}</div>
+        <div class=" collapsible-body" style="display: block" ng-show="showBody1 === 'true'"><p>Job Description: {{job.content}}</p>
+          <p>Creation Date: {{job.created}}</p>
+          <a id="contact-us-button" class="btn waves-effect waves-light blue" ng-click="openCareerApplication();">Apply Now</a>
         </div>
```
Comment (ssk13) [inline modules/core/client/views/careers.client.view.html:39]: get rid of the 'blue' class so that it styles this button like the rest of our buttons

**Comment by ssk13** [comment]
aside from one HTML edit, looks good to me!

**Comment by DannyRL** [comment]
Functionality is all good!

**Pull request merged**